pre_commit_hooks/requirements_txt_fixer.py

```diff
@@ -10,18 +11,31 @@
 
 
 class Requirement:
+    UNTIL_COMPARISON = re.compile(b'={2,3}|!=|~=|>=?|<=?')
```

Comment:
is this supposed to match `===` ?

Reply:
Yes, we match it here too https://github.com/asottile/setup-cfg-fmt/blob/c3a291b4615a1ed07b275dd37d20df9ca02f8880/setup_cfg_fmt.py#L228

Reply:
huh, TIL I guess